src/privateer/cpp/privateer-lib.h

```diff
@@ -489,15 +499,15 @@ namespace privateer
         {
             public:
                 shadedCircle() { } //!< null constructor
-                shadedCircle( int x, int y, std::string message, std::string mmdbsel = "" ) { set_pos(x, y); set_tooltip ( message ); set_mmdbsel(mmdbsel);}
+                shadedCircle( int x, int y, std::string message, std::string chainID = "N/A", std::string resname = "N/A", int seqnum = 0, std::string mmdbsel = "" ) { set_pos(x, y); set_tooltip ( message ); set_mmdbsel(mmdbsel); set_chainID(chainID); set_resname(resname); set_seqnum(seqnum);}
```

Comment:
I think I deliberately excluded shaded nodes from including the metadata, but on other hand I can see the benefit of this in case users don't aim their mouse at the sugar nodes themselves.

Reply:
Did I change something from your PR here? If I did it wasn't intentional

Reply:
My bad, you're right. The change was done by me and I have gone delusional here. Please disregard my comment